kodilla-spring-basic/src/main/java/com/kodilla/spring/basic/spring_configuration/homework/Sedan.java

```diff
@@ -0,0 +1,19 @@
+package com.kodilla.spring.basic.spring_configuration.homework;
+
+import java.time.LocalTime;
+
+public class Sedan implements Car {
+
+    @Override
+    public boolean hasHeadlightsTurnedOn(LocalTime localTime) {
+        if ((localTime.isAfter(LocalTime.of(19, 59, 59)) || localTime.isBefore(LocalTime.of(6, 0)))) {
```

Comment:
Zrob stale dla tych LocalTime - za kazdym razem tworzysz obiekt i dodatkowo we wszystkich klasach tak samo wyglada (w interfejsie mozesz kiec stale przeciez)